integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/container/TrinoITContainers.java

```diff
@@ -49,6 +49,12 @@ public void launch(int gravitinoServerPort) throws Exception {
             command, false, ProcessData.TypesOfData.STREAMS_MERGED, env);
     LOG.info("Command {} output:\n{}", command, output);
 
+    String outputString = output.toString();
+    if (Strings.isNotEmpty(outputString)
+        && !outputString.contains("All docker compose service is now available")) {
```

Comment:
If the log contains `All docker compose service is now available`, why it will throw `The container for the %s service is not started` in following `resolveServerAddress()`?

Reply:
There is a "!" before the logic `outputString.contains("All docker compose service is now available")`.

Reply:
I know, I mean that we may find `All docker compose service is now available` and `The container for the %s service is not started` both in log, is it wired?

Reply:
It is a bit strange, but the difference here is that one checks whether the container IP is functioning properly, while the other checks whether the Docker Compose has started successfully.

Reply:
So the error message should be more accuracy